src/shell/components/global-notifications/GlobalNotifications.js

```diff
@@ -213,6 +213,31 @@ export const CustomNotification = forwardRef(({ id, ...props }, ref) => {
     closeSnackbar(id);
   }, [id, closeSnackbar]);
 
+  const parseStrToBold = (str, isHeading = false) => {
+    if (str.indexOf(":") !== -1) {
+      return (
+        <>
+          <strong>{str.substring(0, str.indexOf(":") + 1)}</strong>
```

Comment:
I would prefer we use MUI's Typography component with a specified `fontWeight` to make it appear bold  instead of `<strong></strong>` just to be consistent.

Reply:
done updating your request @finnar-bin . thank you